dependencies.yaml

```diff
@@ -133,7 +137,6 @@ files:
     includes:
       - test_python_cuproj
       - depends_on_cuspatial
-      - test_cuspatial
```

Comment:
This section populates the `[test]` extra for `cuproj` wheels.

The `test_cuspatial` list contained only `conda` dependencies, so it didn't affect wheels in any way. That's why you see this being removed without replacement.